plugins/ipv4net/ipv4net_api.go

```diff
@@ -28,6 +28,10 @@ type API interface {
 	// GetHostIPs returns all IP addresses of this node present in the host network namespace (Linux).
 	GetHostIPs() []net.IP
 
+	// GetPodSubnetThisNode returns POD network for the current node
+	// (given by nodeID allocated for this node).
+	GetPodSubnetThisNode() *net.IPNet
```

Comment:
Why was this API added to ipv4net plugin? This information is already provided by IPAM and ipv4net is now just forwarding it. Any plugin that needs to know the pod subnet can add IPAM as dependency and read it directly from there.